workers/r_processors/cel_to_pcl.r

```diff
@@ -0,0 +1,40 @@
+ProcessCelFiles <- function(process.dir, organism.code, output.file) {
+    library("affy")
```

Comment:
Which of the functions below comes from these libraries? We typically like to use [double colon operators](https://stat.ethz.ch/R-manual/R-devel/library/base/html/ns-dblcolon.html). I think this is helpful from a readability standpoint (not necessarily always faster than using `library`).